pkg/omf/functions/bundle/omf.bundle.add.fish

```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
+  if test -h $OMF_CONFIG/bundle
```

Comment:
I'm looking both at the fish website and the man page, and `-h` is undocumented. Maybe `-L` would make it easier for people to understand what it's doing?